### PR TITLE
Removed unsupported versions from database products

### PR DESCRIPTION
**Pull request by dwicinas**
Includes changes to templates and generated files for:

PostgreSQL
EDB Postgres Advanced Server
EDB Postgres Extended Server

### CODE REVIEW

File: product_docs/docs/epas/11/installing/linux_install_details/managing_an_advanced_server_installation.mdx [hunk 1]
```diff
@@ -114,7 +114,7 @@ You can use the `pg_ctl` utility to control an Advanced Server service from the
 
 For more information about using the `pg_ctl` utility, or the command line options available, see the official PostgreSQL Core Documentation available at:
```
Comment (nidhibhammar) [inline product_docs/docs/epas/11/installing/linux_install_details/managing_an_advanced_server_installation.mdx:115]:
```suggestion
For more information about using the `pg_ctl` utility, or the command line options available, see the official PostgreSQL Core Documentation available at:
```
```suggestion
For more information about using the `pg_ctl` utility or the command-line options available, see the official [PostgreSQL Core Documentation available](https://www.postgresql.org/docs/11/static/app-pg-ctl.html).
```

File: product_docs/docs/epas/11/installing/linux_install_details/managing_an_advanced_server_installation.mdx [hunk 2]
```diff
@@ -114,7 +114,7 @@ You can use the `pg_ctl` utility to control an Advanced Server service from the
...
 
-<https://www.postgresql.org/docs/11/static/app-pg-ctl.html>
+[https://www.postgresql.org/docs/11/static/app-pg-ctl.html]<https://www.postgresql.org/docs/11/static/app-pg-ctl.html>
```
Comment (nidhibhammar) [inline product_docs/docs/epas/11/installing/linux_install_details/managing_an_advanced_server_installation.mdx:117]:
```suggestion
```

File: product_docs/docs/epas/11/installing/linux_install_details/managing_an_advanced_server_installation.mdx [hunk 3]
```diff
@@ -150,28 +150,28 @@ Where:
 
 For more information about using the command line client, see the PostgreSQL Core Documentation at:
```
Comment (nidhibhammar) [inline product_docs/docs/epas/11/installing/linux_install_details/managing_an_advanced_server_installation.mdx:151]:
```suggestion
For more information about using the command line client, see the [PostgreSQL Core Documentation](https://www.postgresql.org/docs/11/static/app-psql.html).
```

File: product_docs/docs/epas/11/installing/linux_install_details/managing_an_advanced_server_installation.mdx [hunk 4]
```diff
@@ -150,28 +150,28 @@ Where:
...
 
-<https://www.postgresql.org/docs/11/static/app-psql.html>
+[https://www.postgresql.org/docs/11/static/app-psql.html]<https://www.postgresql.org/docs/11/static/app-psql.html>
```
Comment (nidhibhammar) [inline product_docs/docs/epas/11/installing/linux_install_details/managing_an_advanced_server_installation.mdx:153]:
```suggestion
```

File: product_docs/docs/epas/11/installing/linux_install_details/managing_an_advanced_server_installation.mdx [hunk 5]
```diff
@@ -150,28 +150,28 @@ Where:
...
 The PostgreSQL `initdb` command creates a database cluster. After installing Advanced Server, you must manually configure the service and invoke `initdb` to create your cluster. When invoking `initdb`, you can:
 
 -   Specify environment options on the command line.
--   Include the `systemd` service manager on RHEL/CentOS 7.x or RHEL/Rocky Linux/AlmaLinux 8.x and use a service configuration file to configure the environment.
+-   Include the `systemd` service manager on RHEL/Rocky Linux/AlmaLinux 8.x and use a service configuration file to configure the environment.
 
 To review the `initdb` documentation, visit:
 
-<https://www.postgresql.org/docs/11/static/app-initdb.html>
+[https://www.postgresql.org/docs/11/static/app-initdb.html]<https://www.postgresql.org/docs/11/static/app-initdb.html>
```
Comment (nidhibhammar) [inline product_docs/docs/epas/11/installing/linux_install_details/managing_an_advanced_server_installation.mdx:168]:
```suggestion
```

File: product_docs/docs/epas/11/installing/linux_install_details/rpm_packages.mdx [hunk 6]
```diff
@@ -169,36 +163,20 @@ apt list edb*
 
 ### Updating an RPM installation
```
Comment (nidhibhammar) [inline product_docs/docs/epas/11/installing/linux_install_details/rpm_packages.mdx:164]: Do you think we can change this to "Updating a RPM installation"

File: product_docs/docs/epas/11/troubleshooting/linux_troubleshooting.mdx [hunk 7]
```diff
@@ -14,7 +14,7 @@ legacyRedirectsGenerated:
 
 If you encounter difficulty displaying Java-based server features (controls or text not being displayed correctly, or blank windows), upgrading to the latest `libxcb-xlib` libraries should correct the problem on most distributions. Please visit the following link for other possible work-arounds:
```
Comment (nidhibhammar) [inline product_docs/docs/epas/11/troubleshooting/linux_troubleshooting.mdx:15]:
```suggestion
If you encounter difficulty displaying Java-based server features (controls or text not being displayed correctly, or blank windows), upgrading to the latest `libxcb-xlib` libraries should correct the problem on most distributions. Please visit this [link](http://bugs.sun.com/bugdatabase/view_bug.do?bug_id=6532373) for other possible work-arounds.
```

File: product_docs/docs/epas/11/troubleshooting/linux_troubleshooting.mdx [hunk 8]
```diff
@@ -14,7 +14,7 @@ legacyRedirectsGenerated:
...
 
-<http://bugs.sun.com/bugdatabase/view_bug.do?bug_id=6532373>
+[http://bugs.sun.com/bugdatabase/view_bug.do?bug_id=6532373]<http://bugs.sun.com/bugdatabase/view_bug.do?bug_id=6532373>
```
Comment (nidhibhammar) [inline product_docs/docs/epas/11/troubleshooting/linux_troubleshooting.mdx:17]:
```suggestion
```

File: product_docs/docs/epas/11/uninstalling/linux_uninstall.mdx [hunk 9]
```diff
@@ -27,40 +27,24 @@ rpm -e <package_name>
...
-    dnf remove <package_name>
-    ```
+```text
+dnf remove <package_name>
+```
 
 Where `package_name` is the name of the package that you would like to remove.
 
-`yum` and RPM will not remove a package that is required by another package. If you attempt to remove a package that satisfies a package dependency, `yum` or RPM will provide a warning.
+`rpm` doesn't remove a package that is required by another package. If you attempt to remove a package that satisfies a package dependency, `rpm' provides a warning.
```
Comment (nidhibhammar) [inline product_docs/docs/epas/11/uninstalling/linux_uninstall.mdx:38]:
```suggestion
`rpm` doesn't remove a package that another package requires. If you attempt to remove a package that satisfies a package dependency, `rpm` provides a warning.
```

File: product_docs/docs/epas/12/troubleshooting/linux_troubleshooting.mdx [hunk 10]
```diff
@@ -14,7 +14,7 @@ legacyRedirectsGenerated:
 
 If you encounter difficulty displaying Java-based server features (controls or text not being displayed correctly, or blank windows), upgrading to the latest `libxcb-xlib` libraries should correct the problem on most distributions. Please visit the following link for other possible work-arounds:
```
Comment (nidhibhammar) [inline product_docs/docs/epas/12/troubleshooting/linux_troubleshooting.mdx:15]:
```suggestion
If you encounter difficulty displaying Java-based server features (controls or text not being displayed correctly or blank windows), upgrading to the latest `libxcb-xlib` libraries should correct the problem on most distributions. Please visit this [link](http://bugs.sun.com/bugdatabase/view_bug.do?bug_id=6532373) for other possible workarounds.
```

File: product_docs/docs/epas/12/installing/linux_install_details/rpm_packages.mdx [hunk 11]
```diff
@@ -171,34 +165,18 @@ apt list edb*
...
 
     ```text
     dnf upgrade edb*
     ```
 
 !!! Note
-    The `yum upgrade` or `dnf upgrade` command perform an update only between minor releases; to update between major releases, you must use `pg_upgrade`.
+    The `dnf upgrade` command perform an update only between minor releases; to update between major releases, you must use `pg_upgrade`.
 
 For more information about using yum commands and options, enter `yum --help` on your command line.
```
Comment (nidhibhammar) [inline product_docs/docs/epas/12/installing/linux_install_details/rpm_packages.mdx:181]:
```suggestion
For more information about using yum commands and options, enter `dnf --help` on your command line.
```

File: product_docs/docs/epas/12/installing/linux_install_details/managing_an_advanced_server_installation.mdx [hunk 12]
```diff
@@ -117,7 +117,7 @@ You can use the `pg_ctl` utility to control an Advanced Server service from the
 
 For more information about using the `pg_ctl` utility, or the command line options available, see the official PostgreSQL Core Documentation available at:
```
Comment (nidhibhammar) [inline product_docs/docs/epas/12/installing/linux_install_details/managing_an_advanced_server_installation.mdx:118]:
```suggestion
For more information about using the `pg_ctl` utility, or the command line options available, see the official [PostgreSQL Core Documentation](https://www.postgresql.org/docs/12/static/app-pg-ctl.html).
```

File: product_docs/docs/epas/12/installing/linux_install_details/managing_an_advanced_server_installation.mdx [hunk 13]
```diff
@@ -117,7 +117,7 @@ You can use the `pg_ctl` utility to control an Advanced Server service from the
...
 
-<https://www.postgresql.org/docs/12/static/app-pg-ctl.html>
+[https://www.postgresql.org/docs/12/static/app-pg-ctl.html]<https://www.postgresql.org/docs/12/static/app-pg-ctl.html>
```
Comment (nidhibhammar) [inline product_docs/docs/epas/12/installing/linux_install_details/managing_an_advanced_server_installation.mdx:120]:
```suggestion
```

File: product_docs/docs/epas/12/installing/linux_install_details/managing_an_advanced_server_installation.mdx [hunk 14]
```diff
@@ -153,25 +153,25 @@ Where:
 
 For more information about using the command line client, see the PostgreSQL Core Documentation at:
```
Comment (nidhibhammar) [inline product_docs/docs/epas/12/installing/linux_install_details/managing_an_advanced_server_installation.mdx:154]:
```suggestion
For more information about using the command line client, see the [PostgreSQL Core Documentation](https://www.postgresql.org/docs/12/static/app-psql.html).
```

File: product_docs/docs/epas/12/installing/linux_install_details/managing_an_advanced_server_installation.mdx [hunk 15]
```diff
@@ -153,25 +153,25 @@ Where:
...
 
 ## Creating a Database Cluster and Starting the Service
 
 The PostgreSQL `initdb` command creates a database cluster; when installing Advanced Server with an RPM package, the `initdb` executable is in `/usr/edb/asx.x/bin`. After installing Advanced Server, you must manually configure the service and invoke `initdb` to create your cluster. When invoking `initdb`, you can:
 
 -   Specify environment options on the command line.
--   Include the `systemd` service manager on RHEL/CentOS 7.x or RHEL/Rocky Linux/AlmaLinux 8.x and use a service configuration file to configure the environment.
+-   Include the `systemd` service manager on RHEL/Rocky Linux/AlmaLinux 8.x and use a service configuration file to configure the environment.
 
 To review the `initdb` documentation, visit:
```
Comment (nidhibhammar) [inline product_docs/docs/epas/12/installing/linux_install_details/managing_an_advanced_server_installation.mdx:166]:
```suggestion
For more information, see the [initdb documentation](https://www.postgresql.org/docs/12/static/app-initdb.html).
```

File: product_docs/docs/epas/12/installing/linux_install_details/managing_an_advanced_server_installation.mdx [hunk 16]
```diff
@@ -153,25 +153,25 @@ Where:
...
 
-<https://www.postgresql.org/docs/12/static/app-initdb.html>
+[https://www.postgresql.org/docs/12/static/app-initdb.html]<https://www.postgresql.org/docs/12/static/app-initdb.html>
```
Comment (nidhibhammar) [inline product_docs/docs/epas/12/installing/linux_install_details/managing_an_advanced_server_installation.mdx:168]:
```suggestion
```

File: product_docs/docs/epas/12/installing/linux_install_details/managing_an_advanced_server_installation.mdx [hunk 17]
```diff
@@ -308,7 +308,7 @@ Include the `--icu-short-form` keywords to create a cluster that uses a default
 
 For more information about using `initdb`, and the available cluster configuration options, see the PostgreSQL Core Documentation available at:
 
-<https://www.postgresql.org/docs/12/static/app-initdb.html>
+[https://www.postgresql.org/docs/12/static/app-initdb.html]<https://www.postgresql.org/docs/12/static/app-initdb.html>
```
Comment (nidhibhammar) [inline product_docs/docs/epas/12/installing/linux_install_details/managing_an_advanced_server_installation.mdx:311]:
```suggestion
```

File: product_docs/docs/epas/12/troubleshooting/linux_troubleshooting.mdx [hunk 18]
```diff
@@ -14,7 +14,7 @@ legacyRedirectsGenerated:
...
 
-<http://bugs.sun.com/bugdatabase/view_bug.do?bug_id=6532373>
+[http://bugs.sun.com/bugdatabase/view_bug.do?bug_id=6532373]<http://bugs.sun.com/bugdatabase/view_bug.do?bug_id=6532373>
```
Comment (nidhibhammar) [inline product_docs/docs/epas/12/troubleshooting/linux_troubleshooting.mdx:17]:
```suggestion
```

File: product_docs/docs/epas/12/uninstalling/linux_uninstall.mdx [hunk 19]
```diff
@@ -28,40 +28,24 @@ rpm -e <package_name>
...
-    dnf remove <package_name>
-    ```
+```text
+dnf remove <package_name>
+```
 
 Where `package_name` is the name of the package that you would like to remove.
 
-`yum` and RPM will not remove a package that is required by another package. If you attempt to remove a package that satisfies a package dependency, `yum` or RPM will provide a warning.
+`rpm` doesn't remove a package that is required by another package. If you attempt to remove a package that satisfies a package dependency, `rpm` provides a warning.
```
Comment (nidhibhammar) [inline product_docs/docs/epas/12/uninstalling/linux_uninstall.mdx:39]:
```suggestion
`rpm` doesn't remove a package that another package requires. If you attempt to remove a package that satisfies a package dependency, `rpm` provides a warning.
```

File: product_docs/docs/epas/13/installing/linux_install_details/managing_an_advanced_server_installation.mdx [hunk 20]
```diff
@@ -103,7 +103,7 @@ You can use the `pg_ctl` utility to control an Advanced Server service from the
 
 For more information about using the `pg_ctl` utility, or the command line options available, see the official PostgreSQL Core Documentation available at:
```
Comment (nidhibhammar) [inline product_docs/docs/epas/13/installing/linux_install_details/managing_an_advanced_server_installation.mdx:104]:
```suggestion
For more information about using the `pg_ctl` utility or the command line options available, see the official [PostgreSQL Core Documentation](https://www.postgresql.org/docs/current/static/app-pg-ctl.html).
```

File: product_docs/docs/epas/13/installing/linux_install_details/managing_an_advanced_server_installation.mdx [hunk 21]
```diff
@@ -103,7 +103,7 @@ You can use the `pg_ctl` utility to control an Advanced Server service from the
...
 
-<https://www.postgresql.org/docs/current/static/app-pg-ctl.html>
+[https://www.postgresql.org/docs/current/static/app-pg-ctl.html]<https://www.postgresql.org/docs/current/static/app-pg-ctl.html>
```
Comment (nidhibhammar) [inline product_docs/docs/epas/13/installing/linux_install_details/managing_an_advanced_server_installation.mdx:106]:
```suggestion
```

File: product_docs/docs/epas/13/installing/linux_install_details/managing_an_advanced_server_installation.mdx [hunk 22]
```diff
@@ -139,7 +139,7 @@ Where:
 
 `edb-psql` is a symbolic link to a binary called `psql`, a modified version of the PostgreSQL community `psql`, with added support for Advanced Server features. . For more information about using the command line client, see the PostgreSQL Core Documentation at:
```
Comment (nidhibhammar) [inline product_docs/docs/epas/13/installing/linux_install_details/managing_an_advanced_server_installation.mdx:140]:
```suggestion
`edb-psql` is a symbolic link to a binary called `psql`, a modified version of the PostgreSQL community `psql`, with added support for Advanced Server features.  For more information about using the command line client, see the [PostgreSQL Core Documentation](https://www.postgresql.org/docs/current/static/app-psql.html).
```

File: product_docs/docs/epas/13/installing/linux_install_details/managing_an_advanced_server_installation.mdx [hunk 23]
```diff
@@ -139,7 +139,7 @@ Where:
...
 
-<https://www.postgresql.org/docs/current/static/app-psql.html>
+[https://www.postgresql.org/docs/current/static/app-psql.html]<https://www.postgresql.org/docs/current/static/app-psql.html>
```
Comment (nidhibhammar) [inline product_docs/docs/epas/13/installing/linux_install_details/managing_an_advanced_server_installation.mdx:142]:
```suggestion
```

File: product_docs/docs/epas/13/installing/linux_install_details/managing_an_advanced_server_installation.mdx [hunk 24]
```diff
@@ -150,17 +150,17 @@ The packages that install the database server component create service startup s
 The PostgreSQL `initdb` command creates a database cluster; when installing Advanced Server with an RPM package, the `initdb` executable is in `/usr/edb/asx.x/bin`. After installing Advanced Server, you must manually configure the service and invoke `initdb` to create your cluster. When invoking `initdb`, you can:
 
 -   Specify environment options on the command line.
--   Include the `systemd` service manager on RHEL/CentOS 7.x or RHEL/Rocky Linux/AlmaLinux 8.x and use a service configuration file to configure the environment.
+-   Include the `systemd` service manager on RHEL/Rocky Linux/AlmaLinux 8.x and use a service configuration file to configure the environment.
 
 To review the `initdb` documentation, visit:
 
-<https://www.postgresql.org/docs/current/static/app-initdb.html>
+[https://www.postgresql.org/docs/current/static/app-initdb.html]<https://www.postgresql.org/docs/current/static/app-initdb.html>
```
Comment (nidhibhammar) [inline product_docs/docs/epas/13/installing/linux_install_details/managing_an_advanced_server_installation.mdx:157]:
```suggestion
```

File: product_docs/docs/epas/13/installing/linux_install_details/managing_an_advanced_server_installation.mdx [hunk 25]
```diff
@@ -297,7 +297,7 @@ Include the `--icu-short-form` keywords to create a cluster that uses a default
 
 For more information about using `initdb`, and the available cluster configuration options, see the PostgreSQL Core Documentation available at:
```
Comment (nidhibhammar) [inline product_docs/docs/epas/13/installing/linux_install_details/managing_an_advanced_server_installation.mdx:298]:
```suggestion
For more information about using `initdb`, and the available cluster configuration options, see the [PostgreSQL Core Documentation](https://www.postgresql.org/docs/current/static/app-initdb.html).
```

File: product_docs/docs/epas/13/installing/linux_install_details/managing_an_advanced_server_installation.mdx [hunk 26]
```diff
@@ -297,7 +297,7 @@ Include the `--icu-short-form` keywords to create a cluster that uses a default
...
 
-<https://www.postgresql.org/docs/current/static/app-initdb.html>
+[https://www.postgresql.org/docs/current/static/app-initdb.html]<https://www.postgresql.org/docs/current/static/app-initdb.html>
```
Comment (nidhibhammar) [inline product_docs/docs/epas/13/installing/linux_install_details/managing_an_advanced_server_installation.mdx:300]:
```suggestion
```

File: product_docs/docs/epas/13/troubleshooting/linux_troubleshooting.mdx [hunk 27]
```diff
@@ -14,7 +14,7 @@ legacyRedirectsGenerated:
 
 If you encounter difficulty displaying Java-based server features (controls or text not being displayed correctly, or blank windows), upgrading to the latest `libxcb-xlib` libraries should correct the problem on most distributions. Please visit the following link for other possible work-arounds:
```
Comment (nidhibhammar) [inline product_docs/docs/epas/13/troubleshooting/linux_troubleshooting.mdx:15]:
```suggestion
If you encounter difficulty displaying Java-based server features (controls or text not displayed correctly or blank windows), upgrading to the latest `libxcb-xlib` libraries should correct the problem on most distributions. Please visit this [link](http://bugs.sun.com/bugdatabase/view_bug.do?bug_id=6532373) for other possible workarounds.
```

File: product_docs/docs/epas/13/troubleshooting/linux_troubleshooting.mdx [hunk 28]
```diff
@@ -14,7 +14,7 @@ legacyRedirectsGenerated:
...
 
-<http://bugs.sun.com/bugdatabase/view_bug.do?bug_id=6532373>
+[http://bugs.sun.com/bugdatabase/view_bug.do?bug_id=6532373]<http://bugs.sun.com/bugdatabase/view_bug.do?bug_id=6532373>
```
Comment (nidhibhammar) [inline product_docs/docs/epas/13/troubleshooting/linux_troubleshooting.mdx:17]:
```suggestion
```

File: product_docs/docs/epas/13/uninstalling/linux_uninstall.mdx [hunk 29]
```diff
@@ -24,40 +24,24 @@ rpm -e <package_name>
...
-    dnf remove <package_name>
-    ```
+```text
+dnf remove <package_name>
+```
 
 Where `package_name` is the name of the package that you would like to remove.
 
-`yum` and RPM will not remove a package that is required by another package. If you attempt to remove a package that satisfies a package dependency, `yum` or RPM will provide a warning.
+`rpm` does not remove a package that is required by another package. If you attempt to remove a package that satisfies a package dependency, `rpm` provides a warning.
```
Comment (nidhibhammar) [inline product_docs/docs/epas/13/uninstalling/linux_uninstall.mdx:35]:
```suggestion
`rpm` does not remove a package that another package requires. If you attempt to remove a package that satisfies a package dependency, `rpm` provides a warning.
```

File: product_docs/docs/epas/14/installing/linux_install_details/managing_an_advanced_server_installation.mdx [hunk 30]
```diff
@@ -102,7 +102,7 @@ You can use the `pg_ctl` utility to control an EDB Postgres Advanced Server serv
 
 For more information about using the `pg_ctl` utility, or the command line options available, see the official PostgreSQL Core Documentation available at:
```
Comment (nidhibhammar) [inline product_docs/docs/epas/14/installing/linux_install_details/managing_an_advanced_server_installation.mdx:103]:
```suggestion
For more information about using the `pg_ctl` utility or the command-line options available, see the official [PostgreSQL Core Documentation](https://www.postgresql.org/docs/current/static/app-pg-ctl.html). 
```

File: product_docs/docs/epas/14/installing/linux_install_details/managing_an_advanced_server_installation.mdx [hunk 31]
```diff
@@ -102,7 +102,7 @@ You can use the `pg_ctl` utility to control an EDB Postgres Advanced Server serv
...
 
-<https://www.postgresql.org/docs/current/static/app-pg-ctl.html>
+[https://www.postgresql.org/docs/current/static/app-pg-ctl.html]<https://www.postgresql.org/docs/current/static/app-pg-ctl.html>
```
Comment (nidhibhammar) [inline product_docs/docs/epas/14/installing/linux_install_details/managing_an_advanced_server_installation.mdx:105]:
```suggestion
```

File: product_docs/docs/epas/14/installing/linux_install_details/managing_an_advanced_server_installation.mdx [hunk 32]
```diff
@@ -138,15 +138,15 @@ Where:
 
 `edb-psql` is a symbolic link to a binary called `psql`, a modified version of the PostgreSQL community `psql`, with added support for Advanced Server features. For more information about using the command line client, see the PostgreSQL Core Documentation at:
```
Comment (nidhibhammar) [inline product_docs/docs/epas/14/installing/linux_install_details/managing_an_advanced_server_installation.mdx:139]:
```suggestion
`edb-psql` is a symbolic link to a binary called `psql`, a modified version of the PostgreSQL community `psql`, with added support for Advanced Server features. For more information about using the command line client, see the [PostgreSQL Core Documentation](https://www.postgresql.org/docs/current/static/app-psql.html).
```

File: product_docs/docs/epas/14/installing/linux_install_details/managing_an_advanced_server_installation.mdx [hunk 33]
```diff
@@ -138,15 +138,15 @@ Where:
...
 
-<https://www.postgresql.org/docs/current/static/app-psql.html>
+[https://www.postgresql.org/docs/current/static/app-psql.html]<https://www.postgresql.org/docs/current/static/app-psql.html>
```
Comment (nidhibhammar) [inline product_docs/docs/epas/14/installing/linux_install_details/managing_an_advanced_server_installation.mdx:141]:
```suggestion
```

File: product_docs/docs/epas/14/installing/linux_install_details/managing_an_advanced_server_installation.mdx [hunk 34]
```diff
@@ -138,15 +138,15 @@ Where:
...
 
 ### Managing authentication on a Debian or Ubuntu host
 
 By default, the server is running with the peer or md5 permission on a Debian or Ubuntu host. You can change the authentication method by modifying the `pg_hba.conf` file, located under `/etc/edb-as/14/main/`.
 
 For more information about modifying the `pg_hba.conf` file, see the PostgreSQL core documentation available at:
```
Comment (nidhibhammar) [inline product_docs/docs/epas/14/installing/linux_install_details/managing_an_advanced_server_installation.mdx:147]:
```suggestion
For more information about modifying the `pg_hba.conf` file, see the [PostgreSQL core documentation](https://www.postgresql.org/docs/current/auth-pg-hba-conf.html). 
```

File: product_docs/docs/epas/14/installing/linux_install_details/managing_an_advanced_server_installation.mdx [hunk 35]
```diff
@@ -138,15 +138,15 @@ Where:
...
 
-<https://www.postgresql.org/docs/current/auth-pg-hba-conf.html>
+[https://www.postgresql.org/docs/current/auth-pg-hba-conf.html]<https://www.postgresql.org/docs/current/auth-pg-hba-conf.html>
```
Comment (nidhibhammar) [inline product_docs/docs/epas/14/installing/linux_install_details/managing_an_advanced_server_installation.mdx:149]:
```suggestion
```

File: product_docs/docs/epas/14/installing/linux_install_details/managing_an_advanced_server_installation.mdx [hunk 36]
```diff
@@ -157,17 +157,17 @@ The packages that install the database server component create a unit file (on v
 The PostgreSQL `initdb` command creates a database cluster; when installing EDB Postgres Advanced Server with an RPM package, the `initdb` executable is in `/usr/edb/asx.x/bin`. After installing EDB Postgres Advanced Server, you must manually configure the service and invoke `initdb` to create your cluster. When invoking `initdb`, you can:
 
 -   Specify environment options on the command line.
--   Include the `systemd` service manager on RHEL/CentOS 7.x or RHEL/Rocky Linux/AlmaLinux 8.x and use a service configuration file to configure the environment.
+-   Include the `systemd` service manager on RHEL/Rocky Linux/AlmaLinux 8.x and use a service configuration file to configure the environment.
 
 To review the `initdb` documentation, visit:
```
Comment (nidhibhammar) [inline product_docs/docs/epas/14/installing/linux_install_details/managing_an_advanced_server_installation.mdx:162]:
```suggestion
For more information, see the [initdb documentation](https://www.postgresql.org/docs/current/static/app-initdb.html).
```

File: product_docs/docs/epas/14/installing/linux_install_details/managing_an_advanced_server_installation.mdx [hunk 37]
```diff
@@ -157,17 +157,17 @@ The packages that install the database server component create a unit file (on v
...
 
-<https://www.postgresql.org/docs/current/static/app-initdb.html>
+[https://www.postgresql.org/docs/current/static/app-initdb.html]<https://www.postgresql.org/docs/current/static/app-initdb.html>
```
Comment (nidhibhammar) [inline product_docs/docs/epas/14/installing/linux_install_details/managing_an_advanced_server_installation.mdx:164]:
```suggestion
```

File: product_docs/docs/epas/14/installing/linux_install_details/managing_an_advanced_server_installation.mdx [hunk 38]
```diff
@@ -300,7 +300,7 @@ Include the `--icu-short-form` keywords to create a cluster that uses a default
 
 For more information about using `initdb`, and the available cluster configuration options, see the PostgreSQL Core Documentation available at:
```
Comment (nidhibhammar) [inline product_docs/docs/epas/14/installing/linux_install_details/managing_an_advanced_server_installation.mdx:301]:
```suggestion
For more information about using `initdb`, and the available cluster configuration options, see the [PostgreSQL Core Documentation](https://www.postgresql.org/docs/current/static/app-initdb.html).
```

File: product_docs/docs/epas/14/installing/linux_install_details/managing_an_advanced_server_installation.mdx [hunk 39]
```diff
@@ -300,7 +300,7 @@ Include the `--icu-short-form` keywords to create a cluster that uses a default
...
 
-<https://www.postgresql.org/docs/current/static/app-initdb.html>
+[https://www.postgresql.org/docs/current/static/app-initdb.html]<https://www.postgresql.org/docs/current/static/app-initdb.html>
```
Comment (nidhibhammar) [inline product_docs/docs/epas/14/installing/linux_install_details/managing_an_advanced_server_installation.mdx:303]:
```suggestion
```

File: product_docs/docs/epas/14/uninstalling/linux_uninstall.mdx [hunk 40]
```diff
@@ -22,40 +22,26 @@ rpm -e <package_name>
...
-    dnf remove <package_name>
-    ```
+```text
+dnf remove <package_name>
+```
 
 Where `package_name` is the name of the package that you want to remove.
 
-`yum` and RPM don't remove a package that is required by another package. If you attempt to remove a package that satisfies a package dependency, `yum` or RPM provides a warning.
+`rpm` doesn't remove a package that is required by another package. If you attempt to remove a package that satisfies a package dependency, `rpm` provides a warning.
```
Comment (nidhibhammar) [inline product_docs/docs/epas/14/uninstalling/linux_uninstall.mdx:35]:
```suggestion
`rpm` doesn't remove a package that another package requires. If you attempt to remove a package that satisfies a package dependency, `rpm` provides a warning.
```

File: product_docs/docs/epas/16/installing/linux_install_details/managing_an_advanced_server_installation/configuring_a_package_installation.mdx [hunk 41]
```diff
@@ -37,7 +37,7 @@ After creating the cluster, use `systemctl` to start, stop, or restart the servi
 systemctl { start | stop | restart } edb-as-16
 ```
 
-### On Debian 10.x | 11.x or Ubuntu 20.04 | 22.04
+### On 11.x or Ubuntu 20.04 | 22.04
```
Comment (nidhibhammar) [inline product_docs/docs/epas/16/installing/linux_install_details/managing_an_advanced_server_installation/configuring_a_package_installation.mdx:40]:
```suggestion
### On Debian 11.x or Ubuntu 20.04 | 22.04
```

File: product_docs/docs/epas/16/installing/linux_install_details/managing_an_advanced_server_installation/starting_and_stopping_services.mdx [hunk 42]
```diff
@@ -40,9 +40,9 @@ Where:
 -   `restart` to stop and then start the service.
 -   `status` to discover the current status of the service.
 
-## Controlling a service on Debian 10.x | 11.x or Ubuntu 20.04 | 22.04
+## Controlling a service on 11.x or Ubuntu 20.04 | 22.04
```
Comment (nidhibhammar) [inline product_docs/docs/epas/16/installing/linux_install_details/managing_an_advanced_server_installation/starting_and_stopping_services.mdx:43]:
```suggestion
## Controlling a service on Debian 11.x or Ubuntu 20.04 | 22.04
```

File: product_docs/docs/epas/16/installing/uninstalling/linux_uninstall.mdx [hunk 43]
```diff
@@ -13,7 +13,7 @@ legacyRedirects:
 !!! Note
     After uninstalling EDB Postgres Advanced Server, the cluster data files remain intact, and the service user persists. You can manually remove the cluster `data` and service user from the system.
 
-## Uninstalling on RHEL/OL/AlmaLinux/Rocky Linux
+## Uninstalling on RHEL/AlmaLinux/Rocky Linux
 
 You can use variations of the `rpm`, `yum`, or `dnf` command to remove installed packages. Removing a package doesn't damage the EDB Postgres Advanced Server `data` directory.
```
Comment (nidhibhammar) [inline product_docs/docs/epas/16/installing/uninstalling/linux_uninstall.mdx:18]:
```suggestion
You can use  `dnf` command to remove installed packages. Removing a package doesn't damage the EDB Postgres Advanced Server `data` directory.
```

**Comment by dwicinas** [comment]
@nidhibhammar I committed all your suggestions in this PR. Thanks for the review.

File: product_docs/docs/epas/11/installing/linux_install_details/rpm_packages.mdx (hunk 6 above)
Comment (dwicinas) [reply product_docs/docs/epas/11/installing/linux_install_details/rpm_packages.mdx]: I changed wording to "Updating RPM installations"

**Review by nidhibhammar — APPROVED** [review]

**Pull request merged**